src/components/Editor/components/MyWork/components/MyWorkEditorArea/index.js

```diff
@@ -105,13 +102,13 @@ export default class MyWorkEditorArea extends Component {
                                     data-intro="Describe your work by typing directly 
                                         or using the microphone to record an explanation of your work (required)."
                                     aria-label="Use the microphone button or type to explain your work (required)"
-                                    value={this.state.textAreaValue}
-                                    onChange={e => this.setState({ textAreaValue : e.target.value })}
+                                    value={this.props.textAreaValue}
+                                    onChange={value => this.props.textAreaChanged(value.target.value)}
```

Comment:
You can pass no arguments here, and in the method definition use one parameter - event. This value will be available as event.target.value, and it's more readable I think.